app/modules/user.py

```
@@ -0,0 +1,8 @@
+def create_user(username, password):
+    if not username or not password:
+        return False
+    return True
```

Ruff (0.8.2)

Return the condition `not (not username or not password)` directly

Replace with `return not (not username or not password)`

(SIM103)